scan_filter_chain:
    - name: camera_filter
      type: alpha_sensors/CameraFilter 
      params:
          lower_threshold : 0.01
          upper_threshold : 2.5
          fov: 1.02 # TODO : can it really detect things at the outskirt of vision?
